src/documentation/markdown/2017/lar-data-fields.md

```diff
@@ -0,0 +1,378 @@
+  - 3 - Application denied by financial institution 
+  - 4 - Application withdrawn by applicant
+  - 5 - File closed for incompleteness
+  - 6 - Loan purchased by your institution
+
+### [msa_md](#msa_md)
+
+- **Description:** The 5 digit MSA (metropolitan statistical area) or MD (metropolitan division) code. An MSA/MD is an area that has at least one urbanized area of 50,000 or more population.
+- **Values:**
+  - Varying values
+
+### [state\_code](#state_code)
+
+- **Description:** Two-letter state code
+- **Values:**
+  - Varying values
+
+### [county\_code](#county_code)
+
+- **Description:** State-county FIPS code
+- **Values:**
+  - Varying values
+
+### [census\_tract](#census_tract)
+
+- **Description:** 11 digit census tract number
+- **Values:**
+  - Varying values
+
+### [applicant\_ethnicity\_1](#applicant_ethnicity_1)
+
+- **Description:** Ethnicity of the applicant or borrower
+- **Values:**
+  - 1 - Hispanic or Latino
+  - 2 - Not Hispanic or Latino
+  - 3 - Information not provided by applicant in mail, internet, or telephone application
+  - 4 - Not applicable
+
+### [co\_applicant\_ethnicity\_1](#co_applicant_ethnicity_1)
+
+- **Description:** Ethnicity of the first co-applicant or co-borrower
+- **Values:**
+  - 1 - Hispanic or Latino
+  - 2 - Not Hispanic or Latino
+
+### [applicant\_race\_1](#applicant_race_1)
+
+- **Description:** Race of the applicant or borrower
+- **Values:**
+  - 1 - American Indian or Alaska Native
+  - 2 - Asian or Pacific Islander
+  - 3 - Black or African American
+  - 4 - Hispanic
+  - 5 - White
+  - 6 - Information not provided by applicant in mail, internet, or telephone application
+  - 7 - Not applicable
+
+### [applicant\_race\_2](#applicant_race_2)
+
+- **Description:** Race of the applicant or borrower
+- **Values:**
+  - 1 - American Indian or Alaska Native
+  - 2 - Asian or Pacific Islander
+  - 3 - Black or African American
+  - 4 - Hispanic
+  - 5 - White
+  - 6 - Information not provided by applicant in mail, internet, or telephone application
+  - 7 - Not applicable
+
+### [applicant\_race\_3](#applicant_race_3)
+
+- **Description:** Race of the applicant or borrower
+- **Values:**
+  - 1 - American Indian or Alaska Native
+  - 2 - Asian or Pacific Islander
+  - 3 - Black or African American
+  - 4 - Hispanic
+  - 5 - White
+  - 6 - Information not provided by applicant in mail, internet, or telephone application
+  - 7 - Not applicable
+
+### [applicant\_race\_4](#applicant_race_4)
+
+- **Description:** Race of the applicant or borrower
+- **Values:**
+  - 1 - American Indian or Alaska Native
+  - 2 - Asian or Pacific Islander
+  - 3 - Black or African American
+  - 4 - Hispanic
+  - 5 - White
+  - 6 - Information not provided by applicant in mail, internet, or telephone application
+  - 7 - Not applicable
+
+### [applicant\_race\_5](#applicant_race_5)
+
+- **Description:** Race of the applicant or borrower
+- **Values:**
+  - 1 - American Indian or Alaska Native
```

Comment:
for 2017: 
1.American Indian or Alaska Native
2.Asian
3.Black or African American
4.Native Hawaiian or Other Pacific Islander
5.White